galgebra/printer.py

```diff
@@ -822,6 +819,10 @@ def _print_MatrixBase(self, expr):
 
         return out_str
 
+    def _print_Determinant(self, expr):
+        # sympy `uses |X|` by default, we want `det (X)`
+        return r"\det\left ( {}\right )".format(self._print(expr.args[0]))
```

Comment:
These two styles are just two styles, does SymPy has an options for it？

Reply:
No, it does not. We could make a patch upstream to support an option, but for now it's easiest just to add it here.